docs/ipdk-security.md

```diff
@@ -6,7 +6,7 @@ This document captures all the details related to gRPC ports and secure/insecure
 
 ## gRPC ports
 
-IPDK uses a secure-by-default model. The gRPC server will secure ports by default. A user may choose to open insecure ports if needed, but this will be a conscious decision taken by the user at run time.
+The IPDK Networking-Recipe uses a secure-by-default model. The gRPC server will secure ports by default. A user may choose to open insecure ports if needed, but this will be a conscious decision taken by the user at run time.
```

Comment:
"Networking Recipe" should not be hyphenated.

Reply:
Fixed